src/System.Drawing.Common/ref/Configurations.props

```diff
@@ -1,10 +1,14 @@
 ﻿<Project>
   <PropertyGroup>
-    <BuildConfigurations>
+    <PackageConfigurations>
       netcoreapp;
```

Comment:
Please remove version less tfms from Package Configurations. Add netcoreapp3.0 here and move netcoreapp down into Build Configurations.

Reply:
I agree on doing this, but I think this is not part of the scope of this PR. I was going to do it now, but since everything else uses versionless now, then the P2P references in the reference assembly start failing, because they of course don't have a netcoreapp3.0. configuration yet. If I add it to them, they also have P2P, and I will probably end up doing the remapping to the whole repo.

Should we open an issue for these remappings? Or are we using https://github.com/dotnet/corefx/issues/36577?

I want to call out that we should be careful when doing this remapping on if-defed code that use the netcoreapp constant: https://github.com/dotnet/corefx/blob/master/Directory.Build.props#L434

Reply:
I agree this is out of scope. Let's defer this to another PR. I already started with the work here yesterday: https://github.com/dotnet/corefx/pull/36581.